$NetBSD: patch-autoopts_test_defs.in,v 1.1 2026/09/12 12:14:40 wiz Exp $

Give the test programs the libraries configure found.

They link with the compiler, not libtool, so gettext goes unresolved.
Reported upstream as Savannah sr #111451.

--- autoopts/test/defs.in.orig
+++ autoopts/test/defs.in
@@ -153,7 +153,7 @@
     done
     ;;
   esac
-  LIB="${lo_lib} ${LIB}"
+  LIB="${lo_lib} ${LIB} @LIBS@"
 
   AG_L=run_ag\ ao
   agl_opts="-L${top_builddir}/autoopts/tpl"
